### ISSUE TITLE
Black screen when using player

### ISSUE THREAD

**Issue by eGolono** [dularion/streama#202]
I'm running the grails3 beta v1.0.4

When using the player window (the one that's reached by clicking the title card on the dash) I get a black screen with no options and the content doesn't play. However, if I use the "serve" link that is provided in the file manager, it plays just fine.

Not sure if this is related to a [bug in which streama says that there isn't a video file added yet when there actually is one.](http://i.imgur.com/vclKcVP.png?1)

**Comment by dularion** [comment]
Do you see anything in your stacktrace or browser console?

**Comment by eGolono** [comment]
Whoops, forgot to try checking the browser console. Yes, an error does come up in it:
```
TypeError: Cannot read property 'indexOf' of undefined
    at Object.handleWrongBasepathError (streama.unminified.js:30885)
    at streama.unminified.js:32332
    at vendor.unminified.js:23258
    at vendor.unminified.js:27796
    at r.$eval (vendor.unminified.js:29058)
    at r.$digest (vendor.unminified.js:28874)
    at r.$apply (vendor.unminified.js:29166)
    at e (vendor.unminified.js:23551)
    at u (vendor.unminified.js:23749)
    at XMLHttpRequest.w.onload (vendor.unminified.js:23690)(anonymous function) @ vendor.unminified.js:25482(anonymous function) @ vendor.unminified.js:22234(anonymous function) @ vendor.unminified.js:27804$eval @ vendor.unminified.js:29058$digest @ vendor.unminified.js:28874$apply @ vendor.unminified.js:29166e @ vendor.unminified.js:23551u @ vendor.unminified.js:23749w.onload @ vendor.unminified.js:23690
```

**Comment by dularion** [comment]
hopefully this fixes it https://github.com/dularion/streama/releases/tag/v1.0.5

**Comment by dularion** [comment]
fixed in  v1.0.6

**Comment by eGolono** [comment]
That's awesome! Just tested it and it is indeed working now. Thanks again for your great work on this.
